mnefun/_sss.py

```diff
@@ -739,15 +739,15 @@ def _get_fit_data(raw, p=None, prefix='    ', subj=None):
         return None, None, t_window
 
     # Good to do the fits
-    if not op.isfile(count_fname):
+    if not op.isfile(count_fname) or p.overwrite:
```

Comment:
This one I'm not sure about -- there are other things we might want to overwrite / force recomputation of, such as `.pos` files, or `-annot.h5` files. Currently the way to get mnefun to do this is to delete the associated files. Can we keep it that way for now (so to get this behavior, you manually delete the `locs` and `count` files), and discuss at some point how to make it easier to force it to recompute some things?

For example we might want a `recompute` parameter, or probably better, some caching mechanism that automatically detects when some input of interest has changed value since the last time it was run. But this is a much larger endeavor...